workshops/Convert text to speech/README.md

```diff
@@ -0,0 +1,128 @@
+---
+name:'Convert Text to speech.'
+description:'We will convert text to speech using python.'
+author: '@bezlin6mechminerz'
+---
+
+## Final Output.
+
+<img width="1440" alt="Screenshot 2020-09-19 at 12 43 28 PM" src="https://cloud-amkazzxks.vercel.app/93708256-6181ad80-fb52-11ea-9c6b-f1d7534c5520.png">
+
+So we will be converting text to speech using python today.
+
+View a [live demo](https://repl.it/@bezlin/tts#voice.mp3)
+
+View the [final code](https://repl.it/@bezlin/tts#main.py) .
+
+This workshop will take 15 minutes.
+
+You can hear the voice made using the python code [here.](https://repl.it/@bezlin/tts#voice.mp3)
+
+## Getting started
+
+![vedio](https://cloud-1h9458u6z.vercel.app/54_blog_image_13.gif)
+
+We will use python to develop the text to speech code.
+
+### Setting up the environment for Converting text to speech
+
+[Repl.it](https://repl.it) is a simple yet powerful online IDE, Editor, Compiler, Interpreter, and REPL. Code, compile, run, and host in 50+ programming languages. So it is super awesome.
+
+To get started, go to [repl](https://repl.it/languages/Python). Your coding environment will be ready in a few seconds!
+<img width="1440" alt="Screenshot 2020-09-18 at 5 39 06 PM" src="https://cloud-o8q46nkko.vercel.app/screenshot_2020-09-25_at_9.23.06_pm.png">
+
+## Step 2: Get started to code
+
+To convert text to speech we need to import a new Library/API. There are several APIs available to convert text to speech in python. One such APIs is the Google Text to Speech API commonly known as the gTTS API. gTTS is a very easy-to-use tool that converts the text entered, into audio which can be saved as an mp3 file.
+The gTTS API supports several languages including English, Hindi, Tamil, French, German, and many more. The speech can be delivered in any one of the two available audio speeds, fast or slow. However, as of the latest update, it is not possible to change the voice of the generated audio.
+
+For getting this module you need to install the gtts package. Don't worry if you are using repl.it.In repl.it it will be automatically installed when you run the code.
```

Comment:
You should assume folks are using replit here. The grammar is off here/on the next line